client/cypress/support/commands.js

```diff
@@ -17,3 +17,18 @@ Cypress.Commands.add("terminalLog", (violations) => {
 
   cy.task("table", violationData);
 });
+
+Cypress.on("window:before:load", (win) => {
+  cy.stub(win.console, "error", (msg) => {
+    if (msg.includes('No reducer provided for key "app"')) {
+      return null;
```

Comment:
so when null is returned, does it skip over?

Reply:
Yeah it'll ignore this specific error that we currently always have in our console